doc/source/tune/guides/plot_tune-usage.rst

```diff
@@ -0,0 +1,375 @@
+
+.. code-block:: python
+
+    import ray
+    from ray import tune
+    from ray.tune.utils import pin_in_object_store, get_pinned_object
+
+    import numpy as np
+
+    ray.init()
+
+    # X_id can be referenced in closures
+    X_id = pin_in_object_store(np.random.random(size=100000000))
+
+    def f(config, reporter):
+        X = get_pinned_object(X_id)
+        # use X
+
+    tune.run(f)
+
+Stopping Trials
+---------------
+
+You can control when trials are stopped early by passing the ``stop`` argument to ``tune.run``. This argument takes either a dictionary or a function.
+
+If a dictionary is passed in, the keys may be any field in the return result of ``tune.track.log`` in the Function API or ``_train()`` (including the results from ``_train`` and auto-filled metrics).
+
+In the example below, each trial will be stopped either when it completes 10 iterations OR when it reaches a mean accuracy of 0.98. These metrics are assumed to be **increasing**.
+
+.. code-block:: python
+
+    # training_iteration is an auto-filled metric by Tune.
+    tune.run(
+        my_trainable,
+        stop={"training_iteration": 10, "mean_accuracy": 0.98}
+    )
+
+For more flexibility, you can pass in a function instead. If a function is passed in, it must take ``(trial_id, result)`` as arguments and return a boolean (``True`` if trial should be stopped and ``False`` otherwise).
+
+.. code-block:: python
+
+
+    def stopper(trial_id, result):
+        return result["mean_accuracy"] / result["training_iteration"] > 5
+
+    tune.run(my_trainable, stop=stopper)
+
+Finally, you can implement the ``Stopper`` abstract class for stopping entire experiments. For example, the following example stops all trials after the criteria is fulfilled by any individual trial, and prevents new ones from starting:
+
+.. code-block:: python
+
+    from ray.tune import Stopper
+
+    class CustomStopper(Stopper):
+        def __init__(self):
+            self.should_stop = False
+
+        def __call__(self, trial_id, result):
+            if not self.should_stop and result['foo'] > 10:
+                self.should_stop = True
+            return self.should_stop
+
+        def stop_all(self):
+            """Returns whether to stop trials and prevent new ones from starting."""
+            return self.should_stop
+
+    stopper = CustomStopper()
+    tune.run(my_trainable, stop=stopper)
+
+
+Note that in the above example the currently running trials will not stop immediately but will do so once their current iterations are complete. See the :ref:`tune-stop-ref` documentation.
+
+Logging/Tensorboard
```

Comment:
How is this different from the logging page?

Reply:
I wanted to have a section that "previews"/points to the logging page. The logging page is "thick reference material", while this 'usage guide' is more of an overview/tldr.